### PR TITLE
Replaced methods return values from HttpResponse to Pairs

### PR DESCRIPTION
**Pull request by olgamorozova1**

### CODE REVIEW

**Review by EugenBorisik — CHANGES_REQUESTED** [review]
@olgamorozova1 please close response in clients code

File: src/main/java/by/issoft/training/clients/UserClient.java [hunk 1]
```diff
@@ -25,78 +27,70 @@
...
-
-    public CloseableHttpResponse getUsersResponse(ParametersForGetRequest parameters, String sex) {
-        return get.getUsersWithSexParameter("/users", parameters, sex);
+    public Pair<Integer, List<UserDto>> getUsers(ParametersForGetRequest parameters, int age) {
+        return createResponseAndBodyPairFromGetUsersRequest(get.getUsersWithAgeParameter("/users", parameters, age));
     }
 
-    public CloseableHttpResponse getUsersResponse(ParametersForGetRequest parameters, Integer age) {
-        return get.getUsersWithAgeParameter("/users", parameters, age);
+    public Pair<Integer, List<UserDto>> createResponseAndBodyPairFromGetUsersRequest(CloseableHttpResponse response) {
```
[the remark this answers is not in the document]
Comment (olgamorozova1) [reply src/main/java/by/issoft/training/clients/UserClient.java]: Fixed

File: src/main/java/by/issoft/training/clients/UserClient.java [hunk 2]
```diff
@@ -25,78 +27,70 @@
...
+        int responseCode = response.getStatusLine().getStatusCode();
+        HttpEntity responseBodyEntity = response.getEntity();
+        List<UserDto> responseBody = Converter.convertHttpEntityToObject(responseBodyEntity, UserDto.class);
+        Pair<Integer, List<UserDto>> responseCodeAndBody = new MutablePair<>(responseCode, responseBody);
+        return responseCodeAndBody;
```
[the remark this answers is not in the document]
Comment (olgamorozova1) [reply src/main/java/by/issoft/training/clients/UserClient.java]: Fixed

File: src/main/java/by/issoft/training/clients/UserClient.java [hunk 3]
```diff
@@ -25,78 +27,70 @@
...
+        String responseBodyString = null;
         try {
-            responseString = EntityUtils.toString(responseBodyEntity);
+            responseBodyString = EntityUtils.toString(responseBodyEntity);
         } catch (IOException ioException) {
             ioException.printStackTrace();
         }
-        return responseString;
+        Pair<Integer, String> responseCodeAndBody = new MutablePair<>(responseCode, responseBodyString);
+        return responseCodeAndBody;
```
[the remark this answers is not in the document]
Comment (olgamorozova1) [reply src/main/java/by/issoft/training/clients/UserClient.java]: Fixed

File: src/main/java/by/issoft/training/clients/ZipCodesClient.java [hunk 4]
```diff
@@ -27,17 +29,18 @@
...
         }
-        return Converter.convertJsonToObject(listOfZipCodes, String.class);
-    }
 
-    public CloseableHttpResponse getZipCodesResponse() {
-        return get.executeRequest("/zip-codes");
+        int responseCode = response.getStatusLine().getStatusCode();
+        List<String> responseBody = Converter.convertJsonToObject(listOfZipCodes, String.class);
+        Pair<Integer, List<String>> responseCodeAndBody = new MutablePair<>(responseCode, responseBody);
+        return responseCodeAndBody;
```
[the remark this answers is not in the document]
Comment (olgamorozova1) [reply src/main/java/by/issoft/training/clients/ZipCodesClient.java]: Fixed

**Review by EugenBorisik — APPROVED** [review]

**Pull request merged**